service/src/main/java/bio/terra/workspace/service/workspace/WsmApplicationService.java

```diff
@@ -70,25 +70,29 @@ public WsmApplicationService(
 
   public WsmWorkspaceApplication disableWorkspaceApplication(
       AuthenticatedUserRequest userRequest, Workspace workspace, String applicationId) {
-    return commonAbleJob(userRequest, workspace, applicationId, AbleEnum.DISABLE);
+    return commonAbleJob(
+        userRequest, workspace, new ArrayList<String>(List.of(applicationId)), AbleEnum.DISABLE);
   }
 
   public WsmWorkspaceApplication enableWorkspaceApplication(
       AuthenticatedUserRequest userRequest, Workspace workspace, String applicationId) {
-    return commonAbleJob(userRequest, workspace, applicationId, AbleEnum.ENABLE);
+    return commonAbleJob(
+        userRequest, workspace, new ArrayList<String>(List.of(applicationId)), AbleEnum.ENABLE);
```

Comment:
why wrapping in an arrayList?

Reply:
thanks for explaining. i didn't know that. can you add a comment there "wrap in arraylist for JSON serialization."